tests/unit/oom-score-adj.tcl

```diff
@@ -101,23 +101,27 @@ if {$system_name eq {linux}} {
 
         test {CONFIG SET oom score restored on disable} {
             r config set oom-score-adj no
-            set_oom_score_adj 22
-            assert_equal [get_oom_score_adj] 22
+            set custom_oom [expr [get_oom_score_adj] + 1]
+            set_oom_score_adj $custom_oom
+            assert_equal [get_oom_score_adj] $custom_oom
 
             r config set oom-score-adj-values "9 9 9" oom-score-adj yes
-            assert_equal [get_oom_score_adj] [expr 9+22]
+            assert_equal [get_oom_score_adj] [expr 9+$custom_oom]
 
             r config set oom-score-adj no
-            assert_equal [get_oom_score_adj] 22
+            assert_equal [get_oom_score_adj] $custom_oom
         }
 
         test {CONFIG SET oom score relative and absolute} {
-            set custom_oom 9
+            set custom_oom [expr [get_oom_score_adj] + 1]
```

Comment:
that looks odd, we intend to use this variable as relative, but instead of a small positive value we add the base into it.
i think that instead we have to just split this test and use a different `custom_oom` for each step.
i.e. we can keep a value of 9 for the relative one.
and a value of "current+1" for the absolute one.
and then we don't need the range check.

p.s. i'd rather use assert_range so that we get the values printed on failure.